src/gt4sd/frameworks/gflownet/train/qm9.yaml

```diff
@@ -0,0 +1,5 @@
+lr_decay: 10000
```

Comment:
remove local paths, use `pkg_resources` for files we host in the module (add them as package data).

Reply:
I will remove this file. We don't need it.